tensorflow/core/kernels/dml_concat_op.cc

```diff
@@ -227,30 +227,26 @@ class DmlConcatKernel : public DmlKernel {
     CHECK(!tensors.inputs.empty());
 
     auto inputs = GetDmlTensorDescs(tensors.inputs);
-    auto outputs = GetDmlTensorDescs(tensors.outputs);
+    auto scope = dml::Graph(ctx->GetDmlDevice());
 
-    DML_JOIN_OPERATOR_DESC op_specific_desc = {};
-    op_specific_desc.InputCount = inputs.size();
-    op_specific_desc.InputTensors = inputs.data();
-    op_specific_desc.OutputTensor = outputs.data();
-    op_specific_desc.Axis = kNchwDimensionCount - 2;
+    std::vector<dml::Expression> input_tensors;
```

Comment:
(minor) Wasn't there some abseil mini-vector of a fixed size that you used elsewhere?